src/GrpcProtos/capture.proto

```diff
@@ -95,6 +95,22 @@ message IntrospectionScope {
   repeated uint64 registers = 6;
 }
 
+message ApiEvent {
+  int32 pid = 1;
+  int32 tid = 2;
+  uint64 timestamp_ns = 3;
+
+  // The fields below are used to encode orbit_api::EncodedEvent events. This is
+  // done mainly for the performance gain of not having to allocate memory out
+  // of the Arena for strings.
+  fixed64 r0 = 4;
```

Comment:
What does "r" stand for? Raw? I would consider expanding these names even if we know they just stand for row data.

Reply:
have you tried using bytes in please of string? does it improve the performance?

Reply:
Yeah it stands for registers, might not be the best name, but it maps directly to the previous uprobe-based implementation of manual instrumentation. I modified the comment. Concerning the use of bytes, it would improve performance over strings, yes, but not over fixed64. I'd keep the fixed64 fields until we deprecate the old implementation. For now it's quite convenient to have 6 separate 64 bit values to emulate registers. Concerning performance, as you know, there is a bigger investigation going on as to how we can reduce OrbitService's CPU usage. It seems like the constant decoding/encoding of protobufs is taking the majority of the time on the service side, this will probably be my main focus after the manual instrumentation change.

Reply:
@dimitry- `bytes` is internally a heap-allocated `std::string` just like `string`, the only difference is that `string` requires `UTF-8` encoding.